go.mod

```diff
@@ -12,4 +14,47 @@ require (
 	k8s.io/client-go v0.22.4
 )
 
-go 1.16
+require (
```

Comment:
Are there supposed to be two `require` blocks? Can you please run `go mod tidy`?

Reply:
Interesting, thanks for context!